src/Error/JsonApiExceptionRenderer.php

```diff
@@ -16,6 +16,8 @@
  */
 class JsonApiExceptionRenderer extends \Cake\Error\ExceptionRenderer
 {
+    use \Crud\Traits\QueryLogTrait;
```

Comment:
Just fixed in https://github.com/FriendsOfCake/crud/pull/445/commits/eaaf26ca9bb2e2469486a12bbce3e25d5a55e9c3, thanks (again)